src/main/java/core/basesyntax/FigureBehavior.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface FigureBehavior {
+    double getArea();
+
+    void draw();
```

Comment:
separate them into `AreaCalculator` and `Drawable` interfaces to implement Interface Segregation SOLID principle